### DISCUSSION TITLE
Add "Retry" button to Failed Deliveries.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#522] [Feature Requests]
As https://github.com/anonaddy/anonaddy/issues/520#issue-1903677477 states:

> https://app.addy.io/failed-deliveries

**Comment by w3lld0ne** [comment]
+1
why this issue was converted to discussion? the idea is great.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/anonaddy/anonaddy/discussions/522#discussioncomment-8234058

@w3lld0ne, I don't know. Thanks for the support.

**Comment by willbrowningme** [comment]
> +1 why this issue was converted to discussion? the idea is great.

Because it's a feature request not a bug/issue with the web application.

> If you have a feature that you'd like to see added to addy.io then please feel free to post it on [GitHub discussions](https://github.com/anonaddy/anonaddy/discussions). Make sure to check the [Roadmap](https://github.com/anonaddy/anonaddy/projects/1) first to see if it isn't already in the pipeline.

- From https://addy.io/contact/

If I find out a good way to implement this I will add it.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/anonaddy/anonaddy/discussions/522#discussioncomment-8235230

@willbrowningme, https://github.com/anonaddy/anonaddy/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen+Feature+Request+created%3A*..2024-08-11 would appear to demonstrate that such organisation isn't consistently applied:

> ![image](https://github.com/user-attachments/assets/87fa4e3f-0362-46d5-9322-da77ec9d023b)

**Comment by willbrowningme** [comment]
This has now been added to failed deliveries, see the "Resend" button next to "Download" and "Delete".

**Reply by roke-julian-lockhart** [reply]
@willbrowningme, thanks! I'd mark this as the solution, but I don't have the button available for this thread. Weird. [^1]

[^1]: [`github.com/orgs/community/discussions/2903#discussioncomment-14528655`][1]

[1]: https://github.com/orgs/community/discussions/2903#discussioncomment-14528655

**Reply by willbrowningme** [reply]
I'ts because it is not in the Q&A category.
